ssl/quic/quic_channel.c

```diff
@@ -237,8 +238,10 @@ static int ch_init(QUIC_CHANNEL *ch)
     tls_args.s                          = ch->tls;
     tls_args.crypto_send_cb             = ch_on_crypto_send;
     tls_args.crypto_send_cb_arg         = ch;
-    tls_args.crypto_recv_cb             = ch_on_crypto_recv;
-    tls_args.crypto_recv_cb_arg         = ch;
+    tls_args.crypto_recv_rcd_cb         = ch_on_crypto_recv_record;
+    tls_args.crypto_recv_rcd_cb_arg     = ch;
+    tls_args.crypto_release_rcd_cb      = ch_on_crypto_release_record;
+    tls_args.crypto_release_rcd_cb_arg  = ch;
```

Comment:
Six copies of the `ch` channel pointer seems excessive.

Reply:
This is just continuing the existing pattern (we add one extra callback in this PR). Each callback has its own arg. I am ambivalent about this - I would prefer to say this outside of the scope of this PR.